requirements.txt

```diff
@@ -9,5 +9,5 @@ pandas>=1.1.0
 jsonargparse[signatures]>=3.17.0, <=4.9.0
 click>=7.1.2
 protobuf<=3.20.1
-fsspec
+fsspec==2022.11.0
```

Comment:
Can we have it rather as min version?

Reply:
```suggestion
fsspec >=2022.11.0
```

Reply:
@Borda, I am worried that this behavior of `fsspec.open` can be fixed in the future, thus breaking the fix in `lightning-flash` 🤔 We can set it as a min version and hope that they'll maintain backward compatibility.

Reply:
a see that PT has not compatible dependencies:
https://github.com/Lightning-AI/lightning/blob/caa3329b79965f7712e49bd9f8933f7a4686c49b/requirements/pytorch/base.txt#L8